//package core.io.protocol;
message DceArenaOpt
{
	optional int32	type = 1;//1数据
}
